@xen-orchestra/web-core/lib/components/ui/input/UiTextarea.vue

```diff
@@ -0,0 +1,118 @@
+<!-- v2 -->
+<template>
+  <div class="ui-textarea">
+    <VtsLabel v-if="slots.default" :accent="labelAccent" :required :icon :href><slot /></VtsLabel>
+    <textarea v-model="model" :class="accent" :disabled class="textarea" v-bind="$attrs" />
+    <VtsInfo v-if="slots.info" :accent><slot name="info" /></VtsInfo>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import VtsInfo from '@core/components/info/VtsInfo.vue'
+import VtsLabel from '@core/components/input/VtsLabel.vue'
+import type { IconDefinition } from '@fortawesome/fontawesome-common-types'
+import { computed } from 'vue'
+
+defineOptions({
+  inheritAttrs: false,
+})
+
+const props = defineProps<{
+  accent: 'brand' | 'warning' | 'danger'
+  disabled?: boolean
+  href?: string
+  icon?: IconDefinition
+  required?: boolean
+}>()
+
+const model = defineModel<string>({ required: true })
+
+const slots = defineSlots<{
+  default?(): any
+  info?(): any
+}>()
+
+const labelAccent = computed(() => (props.disabled ? 'brand' : props.accent))
+</script>
+
+<style lang="postcss" scoped>
+.ui-textarea {
+  display: flex;
+  flex-direction: column;
+  gap: 0.4rem;
+
+  .textarea {
+    border-radius: 0.4rem;
+    border-width: 0.1rem;
+    height: 8rem;
+    outline: none;
+    padding: 0.8rem 1.6rem;
+    width: 100%;
+
+    &:disabled {
+      background-color: var(--color-neutral-background-disabled);
+      border-color: var(--color-neutral-border);
+    }
+
+    &.brand:not(:disabled) {
+      border-color: var(--color-neutral-border);
+
+      &:is(:hover, :focus-visible) {
+        border-color: var(--color-normal-item-hover);
+      }
+      &:active {
+        border-color: var(--color-normal-item-active);
+      }
+      &:focus {
```

Comment:
From what I see in the DS, shouldn't the `:focus`  be `.selected` class ? I don't know if this is going to change or not.

Reply:
Maybe @OlivierFL can confirm, but from what i understand:
`.selected` is used in case the element needs to keep its `:focus` style even when it is not actually focused. It become an interaction for the component. (E.g. Menu, Tab,...). 
In the case of the input, i don't see any case where we need to keep the style `:focus` without having a focus on the input

Reply:
It depends on how the browser handles `:focus-visible` right?.
BTW i tried and `Chrome` and `Firefox` and also got the `:focus-visible` when using mouse click :thinking:

Reply:
From what I understand, `:focus` selects something with the mouse and `:focus-visible` with the keyboard (at least with an element like a button)

Reply:
So, for inputs `focus` and `focus-visible` seems to act the sameway. Which pseudo class to use?

Reply:
@OlivierFL  ? :D

Reply:
I'm on it, testing to find the best combination

Reply:
So, from what I understood [here](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible#comparing_focus_and_focus-visible) and in other articles, in the case of an input, we can stick to styling the `:focus` state, since the browser will apply styles when activating the element on both keyboard and mouse click.
I think it doesn’t make sense to apply an `outline` in addition to the border.
The case will be different when dealing with buttons and links.